frost-core/src/keys/refresh.rs

```diff
@@ -114,3 +118,260 @@ pub fn refresh_share<C: Ciphersuite>(
 
     Ok(new_key_package)
 }
+
+/// Part 1 of refresh share with DKG. A refreshing_key is generated and a new package and secret_package are generated.
+/// The identity commitment is removed from the packages.
```

Comment:
Nitpick: This is an implementation detail, I don't think it needs to be part of the public documentation